api-specifications/admin-api/admin-api-1.yaml

```diff
@@ -0,0 +1,1355 @@
+openapi: 3.0.1
+info:
+  title: Admin API Documentation
+  version: v1
+paths:
+  /v1/vendors:
+    get:
+      tags:
+        - Vendors
+      summary: Retrieves all vendors.
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+              schema:
+                required:
+                  - length
+                  - longLength
+                  - rank
+                  - syncRoot
+                  - isReadOnly
+                  - isFixedSize
+                  - isSynchronized
+                type: array
+                items:
+                  $ref: >-
+                    #/components/schemas/EdFi.Ods.AdminApi.Features.Vendors.VendorModel
```

Comment:
As a generic specification for others to follow, please simplify the name above. You don't need to change the C# code at all - just make this version of the Open API specification more generic. For example, you could change `EdFi.Ods.AdminApi.Features.Vendors.VendorModel` to `vendor` everywhere in this file.